src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -103,7 +104,13 @@ public SpriteMapView(GameInstance game)
     // Set the view to show the whole map, if possible
     mapViewWidth = SpriteLibrary.baseSpriteSize * game.gameMap.mapWidth;
     mapViewHeight = SpriteLibrary.baseSpriteSize * game.gameMap.mapHeight;
-    SpriteOptions.setScreenDimensions(mapViewWidth * SpriteOptions.getDrawScale(), mapViewHeight * SpriteOptions.getDrawScale());
+
+    if( !Driver.getInstance().isMaximized() )
+    {
+      SpriteOptions.setScreenDimensions( mapViewWidth  * SpriteOptions.getDrawScale(),
+                                         mapViewHeight * SpriteOptions.getDrawScale() );
+      Driver.getInstance().updateView();
+    }
```

Comment:
Amazing. Completely unrelated to unit movement, but amazing.

How'd you track this one down? It looks like the behavior should be mostly the same in the non-maximized case, which is incidentally also where it would go crazy. I'm assuming `Driver.updateView()` was indirectly invoking `SpriteOptions.setScreenDimensions()` or something silly?

Reply:
Oh. The maximized check is just so starting a map with the game maximized doesn't un-maximize the game on you (I noticed this behavior seeming to bother some users and never bothered to write it down because I'm a silly goose)

I probably bundled it and the fix into one commit because I was testing the changes together.

Reply:
The real fix, for what it's worth, is that all of our `IView`s don't end up calling `Driver.updateView()`.
That function invokes the API to resize the view, which is a questionable plan when you just got an update *from* the API.

I don't know why it wasn't broken earlier, but mostly I just looked at the code and decided that seemed like the right thing to tweak.

Reply:
The resize jankiness has been an issue for... ever, I think (at least on Linux). So good job tracking it down.

Reply:
Huh. I think I only started noticing it after I got a second monitor, and I never had it on Windows.
I think I did get the game to *crash* on Windows by fiddling with the resizing, but that only ever happened once.

Thanks.